app/services/spotlight/resources/oaipmh_builder.rb

```diff
@@ -18,33 +19,36 @@ def to_solr
         harvests = resource.oaipmh_harvests
         resumption_token = harvests.resumption_token
         last_page_evaluated = false
+        start = 0
```

Comment:
This is a debugging line that should be removed

Reply:
thanks for catching this, I've removed it.